# Assignment 2

## Probability with Two Dice

Two fair and unbiased dice are rolled simultaneously. This notebook computes:
- Total number of outcomes
- Probability of getting both faces the same

### Step 1: Total Outcomes
Each die has 6 faces, so total outcomes = 6 × 6 = 36.

In [1]:

# Total outcomes
total_outcomes = 6 * 6
total_outcomes


36

### Step 2: Favorable Outcomes
Outcomes where both dice show the same number:
(1,1), (2,2), (3,3), (4,4), (5,5), (6,6)

In [2]:

# Favorable outcomes
same_face_outcomes = 6
same_face_outcomes


6

### Step 3: Probability Calculation

In [3]:

# Probability calculation
probability = same_face_outcomes / total_outcomes
probability


0.16666666666666666

### Final Answer
Probability of getting both faces the same is **1/6 ≈ 0.1667**